samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitor/Program.cs

```diff
@@ -13,12 +13,7 @@ static async Task Main()
         endpointConfiguration.UsePersistence<InMemoryPersistence>();
         endpointConfiguration.SendFailedMessagesTo("error");
 
-        var transport = endpointConfiguration.UseTransport<MsmqTransport>();
-        var routing = transport.Routing();
-        routing.RegisterPublisher(
-            typeof(ServiceControl.Contracts.MessageFailed).Assembly,
-            "Particular.ServiceControl"
-        );
+        var transport = endpointConfiguration.UseTransport<LearningTransport>();
```

Comment:
Do we need the variable `transport` here or do we keep it around just for potential future use?

Reply:
I don't mind it being there in samples so people don't get the idea it returns void.